flink/flink-base/src/main/java/io/tidb/bigdata/flink/tidb/TiDBBaseDynamicTableSource.java

```diff
@@ -55,4 +85,9 @@ protected String getRequiredProperties(String key) {
     return Preconditions.checkNotNull(properties.get(key), key + " can not be null");
   }
 
+  private long getPropertyLong(String key, long defaultValue) {
```

Comment:
Looks like this method is not being used at all

Reply:
Yes, I will delete this method.